src/Orchard.Web/Modules/Orchard.MediaLibrary/Controllers/AdminController.cs

```diff
@@ -149,12 +175,13 @@ public class AdminController : Controller {
 
         [Themed(false)]
         public ActionResult MediaItem(int id, string displayType = "SummaryAdmin") {
-            var contentItem = Services.ContentManager.Get(id, VersionOptions.Latest);
+            var contentItem = Services.ContentManager.Get(id, VersionOptions.Latest).As<MediaPart>();
```

Comment:
Can we do Get<MediaPart>(id, VersionOptions.Latest) ? (I forgot if the api allows that)